clueless/static/js/controllers.js

```diff
@@ -200,9 +207,23 @@ class GameHub {
         this.gameState.players[i].character.position = message.location;
       }
     }
+    // I don't think a toast is needed for this since you can see the move happen, but we could always add one.
   }
   handleMsgSuggestion(message) {
-    this.gamePanel.showToast('Suggestion: ' + JSON.stringify(message));
+    // Get which player has won by character id
```

Comment:
> has won
Is this comment incorrect, or is there something related to endgame here that I'm not seeing?

Reply:
Comment is incorrect, it was a copy-paste error. Updating